<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortAdj</class>
 <widget class="QDialog" name="ShortAdj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Russian Short Form Adjective</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../usr/share/machinetrans/resources/logo.png</normaloff>../../../../usr/share/machinetrans/resources/logo.png</iconset>
  </property>
  <widget class="LineEditRU" name="rufemEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ruplurLbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Plural</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="ruplurEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>780</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Russian Short Form Adjective for </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="runuetEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="rumascEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rufemLbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Feminine</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rumascLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Masculine</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="runuetLbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nueter</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBttn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>520</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptBttn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>520</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept.</string>
   </property>
  </widget>
  <widget class="LineEditRU" name="stemEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="engLbl">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>English</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="stemLbl">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>330</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stem</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="helpBttn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="LineEditEN" name="enEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="backBttn">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>520</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="shortCheck">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Short Form</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditRU</class>
   <extends>QLineEdit</extends>
   <header>machinetrans.lineeditru</header>
  </customwidget>
  <customwidget>
   <class>LineEditEN</class>
   <extends>QLineEdit</extends>
   <header>machinetrans.lineediten</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>helpBttn</tabstop>
  <tabstop>rumascEdit</tabstop>
  <tabstop>rufemEdit</tabstop>
  <tabstop>runuetEdit</tabstop>
  <tabstop>ruplurEdit</tabstop>
  <tabstop>stemEdit</tabstop>
  <tabstop>enEdit</tabstop>
  <tabstop>quitBttn</tabstop>
  <tabstop>acceptBttn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
